test/studies/shootout/meteor/kbrady/meteor-parallel.chpl

```diff
@@ -300,9 +300,9 @@ proc fillContiguousSpace(board, indx) {
 proc firstEmptyCell(cell, in min) {
   while (min == cell[0] || min == cell[1] ||
          min == cell[2] || min == cell[3] ||
-         min == cell[4]) {
+         min == cell[4]) do
```

Comment:
No, you had it right the first time, I was just confusing about where I put the note (as I foolishly assumed Ben and I'd caught all the single line ones)